hooks/test_post_gen_project.py

```diff
@@ -48,3 +48,15 @@ def test_copy_journal_template_files_bad(
         match=errormessage,
     ):
         copy_journal_template_files(input, project_dir)
+
+
+# C1: valid source dir and target dir.
+#  Expect all files in source dir are copied to target dir.
+def test_copy_all_files():
+    pass
+
+
+# C1: invalid source dir and valid target dir.
```

Comment:
Please can you make this more precious? What are the different invalids that you want to test? You can copy this from our old function basically